html/webappapis/dynamic-markup-insertion/opening-the-input-stream/reload.window.js

```diff
@@ -0,0 +1,53 @@
+// This test tests for the nonexistence of a reload override buffer, which is
```

Comment:
Name should be -historical, I think.

Reply:
In that case most tests I added should be -historical…

Reply:
Fair enough, we can leave it as-is.